src/renderer/component/link/view.jsx

```diff
@@ -26,85 +26,94 @@ type Props = {
   noUnderline: ?boolean,
   description: ?string,
   secondary: ?boolean,
+  type: string
 };
 
-const Button = (props: Props) => {
-  const {
-    onClick,
-    href,
-    title,
-    label,
-    icon,
-    iconRight,
-    disabled,
-    children,
-    navigate,
-    navigateParams,
-    doNavigate,
-    className,
-    inverse,
-    alt,
-    circle,
-    flat,
-    fakeLink,
-    description,
-    noStyle,
-    noUnderline,
-    secondary,
-    ...otherProps
-  } = props;
+class Button extends React.PureComponent<Props> {
+  static defaultProps = {
+    type: "button"
+  }
 
-  const combinedClassName = classnames(
-    'btn',
-    noStyle
-      ? 'btn--no-style'
-      : {
-          'btn--link': fakeLink,
-          'btn--primary': !alt && !fakeLink && !secondary, // default to primary
-          'btn--secondary': secondary,
-          'btn--alt': alt,
-          'btn--inverse': inverse,
-          'btn--disabled': disabled,
-          'btn--circle': circle,
-          'btn--flat': flat,
-          'btn--no-underline': fakeLink && noUnderline,
-          'btn--external-link': href
-        },
-    className
-  );
+  render() {
+    const {
+      onClick,
+      href,
+      title,
+      label,
+      icon,
+      iconRight,
+      disabled,
+      children,
+      navigate,
+      navigateParams,
```

Comment:
Just your normal 22 parameter function. Likely out of scope for this PR, but is it possible some `Button` usage should simply be another component?

Reply:
Agreed. Thinking about what can be removed